ee/tabby-webserver/src/service/db.rs

```diff
@@ -167,25 +167,38 @@ impl DbConn {
         email: String,
         password_encrypted: String,
         is_admin: bool,
-    ) -> Result<()> {
+    ) -> Result<i32> {
         let res = self
             .conn
             .call(move |c| {
-                c.execute(
+                let mut stmt = c.prepare(
                     r#"INSERT INTO users (email, password_encrypted, is_admin) VALUES (?, ?, ?)"#,
-                    params![email, password_encrypted, is_admin],
-                )
+                )?;
+                let id = stmt.insert((email, password_encrypted, is_admin))?;
+                Ok(id)
             })
             .await?;
         if res != 1 {
```

Comment:
This `if` check should be removed since the meaning of `res` has changed from `number of rows` to `user id`

Reply:
nice catch, fixed